game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/ForumPosterEditor.java

```diff
@@ -64,6 +73,43 @@
           SwingComponents.highlightLabelIfNotValid(viewModel.isForumUsernameValid(), usernameLabel);
           SwingComponents.highlightLabelIfNotValid(viewModel.isForumPasswordValid(), passwordLabel);
           testForum.setEnabled(viewModel.isTestForumPostButtonEnabled());
+          helpMessage.setText(
+              viewModel.getForumSelection().equals(NodeBbForumPoster.TRIPLEA_FORUM_DISPLAY_NAME)
```

Comment:
Ditto on view model should be told what to render as opposed to computing what to render. This allows for the logic to be tested as well, you can append to the tests to verify that the right help message is provided under the correct conditions.

Reply:
Updated to move the check to the view model.